docker/uwsgi.ini

```diff
@@ -0,0 +1,33 @@
+# mysite_uwsgi.ini file
+[uwsgi]
+base            = /code
+chdir           = %(base)
+module          = olympia.wsgi
```

Comment:
Production uses `module = olympia.wsgi:application`. Maybe it'll work in your local environment even without `:application`?

Reply:
Yeah, uwsgi defaults to `:application` afaik but I'll add it for explicitness too. Thanks for pointing it out.